examples.md

```diff
@@ -373,3 +374,14 @@ When dependencies are installed later in the workflow, we must specify the same
     restore-keys: |
       ${{ runner.os }}-pods-
 ```
+
+## Swift - Swift Package Manager
+
+```yaml
+- uses: actions/cache@v1
+  with:
+    path: .build
```

Comment:
Is saving the entire `.build` directory recommended?

Reply:
Swift places all build files directly in that folder.
That's also how IBMs swift-buildpack does caching (https://github.com/IBM-Swift/swift-buildpack#caching-of-the-build-directory)

Reply:
@123FLO321 Have you found this caching works for built files or artefacts? https://bugs.swift.org/browse/SR-11760 is being used to track an issue related to how SwiftPM tracks files for the build cache. There was a recent PR for this that was not merged: https://github.com/apple/swift-package-manager/pull/2774.

In my testing (see https://github.com/JosephDuffy/actions-playground/runs/954106324?check_suite_focus=true vs https://github.com/JosephDuffy/actions-playground/runs/954129569?check_suite_focus=true) I see the checkouts cache being used, but the build cache is not being used at all. This is true on stable Xcode (11.5) as was as the 12 beta.

Reply:
@JosephDuffy the Problem there is that no Package.resolved file exists when trying to load the cache. Try adding a `swift package resolve` step before loading cache.

Reply:
@123FLO321 The checkout step has completed and there's a Package.resolved file in that branch (https://github.com/JosephDuffy/actions-playground/compare/swiftpm-cache#diff-d29466d627f33a3d61709bad13c5eadd). It's a cache hit (see the second link) but SwiftPM still performs a rebuild.

The links are a little long/unclear so I'll update the comment to be clearer.